org.vergecurrency.verge-qt.json

```diff
@@ -2,7 +2,7 @@
   "id": "org.vergecurrency.verge-qt",
   "runtime": "org.freedesktop.Platform",
   "sdk": "org.freedesktop.Sdk",
-  "runtime-version": "19.08",
+  "runtime-version": "21.08",
   "command": "verge-qt",
   "finish-args": [
     /* Wayland and Xorg support */
```

Comment:
fallback-x11 please

Reply:
Thanks, just didn't test that it was working. But now I did and it doesn't support wayland:

```
$ flatpak run --nosocket=x11 --env=QT_QPA_PLATFORM=wayland org.vergecurrency.verge-qt

This application failed to start because it could not find or load the Qt platform plugin "wayland"
in "".

Available platform plugins are: xcb.

Reinstalling the application may fix this problem.
```

Reply:
then wayland should be removed.